package.json

```diff
@@ -29,6 +29,11 @@
   "scripts": {
     "lint": "eslint ./src ./test",
     "pretest": "yarn lint",
+    "postrelease:github": "rm .npmrc",
+    "postrelease:npm": "rm .npmrc",
+    "release": "yarn release:github && yarn release:npm",
```

Comment:
Do we want to run the `postrelease` commands too after this?

Reply:
postrelease:github and postrelease:npm deal with the cleanup already :D